ddl/export_prod_data.hql

```diff
@@ -10,20 +10,27 @@
 --   * Field delimiter: "\t"
 --   * Null value for missing recommendations 
 --	(image_id, confidence_rating, source fields): ""
+--   * found_on: list of wikis delimited by ','
 -- 
 -- Changelog:
 --   * 2021-03-08: schema and format freeze.
--- 
+--   * 2021-03-25: append found_on column
+
 use ${hiveconf:username};
 set hivevar:null_value="";
+set hivevar:found_on_delimiter=",";
+set hive.cli.print.header=true;
 
+insert overwrite local directory '${hiveconf:output_path}'
```

Comment:
@clarakosi this change (and related changes in `publish.sh`, where the script is invoked) were necessary to mitigate Parquet logs spilling into Hive's stdout we discussed offline.

`insert overwrite local directory` writes to local disk using a file descriptor other `stdout`, and the Parquet noise does not affect it.

Reply:
Got it!